01-intro-to-R.Rmd

```diff
@@ -265,7 +265,7 @@ animals
 ```
 
 The quotes around "mouse", "rat", etc. are essential here. Without the quotes R
-will assume there is a variable called mouse, rat, etc.
+will assume there are objects called mouse, rat and dog and launch and error message as it can not find them.
```

Comment:
... will assume there are objects called mouse, rat, and dog. As these objects don't exist, R will produce an error message.